### ISSUE TITLE
Unable to redirect or swap devices and app closes on Input Remapping

### ISSUE THREAD

**Issue by Khris777** [matzman666/OpenVR-InputEmulator#201]
I'm using a Vive Pro Wireless on Windows 10 and have three extra trackers paired.

I'm on the current release branch of SteamVR and earlier downloaded and installed the emulator. It shows up fine in the SteamVR.

Now I'm in VR with two of the trackers activated. They show up in the tracker list on SteamVR and their silhouettes are shown in VR as well. 

In the emulator I'm trying to swap (or redirect, whichever does thejob) trackers and controllers since I want to try the Feet Saber mod in Beat Saber with trackers at my ankles.

Problem is I can't do anything. When I select the tracker and try to swap it with the controller I'm getting an error saying "Device not found" even though it's in the list. When I select the controller and try to swap it with the tracker I'm getting "Unknown Error".

I hadn't used the trackers for a while and today when I reconnected them I put the USB dongles into different ports than during the first time when I paired them. But since they show up fine in SteamVR and are tracking I didn't think I would have any issues.

Do you think I have to fully pair my trackers again?

Additionally when I click on Input Remapping in VR the app suddenly closes and vanishes, the button in the overlay vanishes, like as if the app just crashed. When I do it in the desktop I just get the window without any mappings but the app doesn't close/crash.

**Comment by Wanegain** [comment]
Hello, it comes from a recent update of SteamVR... We all have the same issue :-(

**Comment by Khris777** [comment]
Oh no. Thanks for the answer.

Have you found a way to fix it yet?

**Comment by Wanegain** [comment]
Yes, I found it this morning here : https://github.com/matzman666/OpenVR-InputEmulator/issues/198#issuecomment-695218855
